### ISSUE TITLE
Rules not showing up in listview on french language OS

### ISSUE THREAD

**Issue by michellabbe** [zmjack/PortProxyGUI#8]
On french OS, PortProxyGUI doesn't seem to be able to parse `netsh` `show all` command properly and the listview remains empty:

![image](https://user-images.githubusercontent.com/16291382/135152157-34991f00-5069-47b7-a9bd-a6d682746cc4.png)

I'm guessing the issue doesn't happen on all foreign languages because screenshots in issue #2 shows it seem to work fine.

Reproduced with v1.1.1, NET45 and Core31 x64 versions, on multiple computers (Win10 x64 Canadian-french 20H2 and 21H1, Windows Server 2016 French).

Creating a new rule won't show up in listview either, but I can see it worked when checking via command line.

netsh under french OS:
```
C:\windows\system32>netsh interface portproxy show all

Écouter sur ipv4 :             Connecter à ipv4 :

Adresse         Port        Adresse         Port
--------------- ----------  --------------- ----------
172.30.195.57   443         172.30.195.130  443
*               4344        172.30.195.130  4344
```

**Comment by zmjack** [comment]
Thanks for your feedback.

This should now be fixed in the latest version (v1.1.2).

Let me know how you get along. :smiley:

**Comment by michellabbe** [comment]
v1.1.2 works fine.

Thanks for the quick fix!
